modules/features/org.wso2.identity.jaggery.apps.feature/src/main/resources/portal/gadgets/user_auth_apps/js/gadget.js

```diff
@@ -121,7 +121,51 @@ function itemRemoveValidate(appName) {
         itemRemove(appName);
     }, cancelCallback:function () {
     }});
-} 
+}
+
+function reloadGrid() {
+    $.ajax({
+        url: "/portal/gadgets/user_auth_apps/index.jag",
+        type: "POST",
+        data: {
+            cookie: cookie,
+            user: userName
+        },
+        success: function (data) {
+            resp = $.parseJSON(data);
+            if (resp.success == true) {
+                json = resp.data;
+                drawPage();
+            } else {
+                if (typeof resp.reLogin != 'undefined' && resp.reLogin === true) {
+                    window.top.location.href = window.location.protocol + '//' + serverUrl + '/dashboard/logout.jag';
+                } else {
+                    if (resp.message != null && resp.message.length > 0) {
+                        message({
+                            content: resp.message,
+                            type: 'error',
+                            cbk: function () {}
+                        });
+                    } else {
+                        message({
+                            content: 'Error occurred while loading values for the grid.',
+                            type: 'error',
+                            cbk: function () {}
+                        });
+                    }
+                }
+            }
+
```

Comment:
Unnecessary blank line

Reply:
FIXED : https://github.com/wso2/product-is/pull/7242

Reply:
FIXED : https://github.com/wso2/product-is/pull/7242